subsys/mpsl/cx/Kconfig

```diff
@@ -61,7 +67,7 @@ config MPSL_CX_GENERIC_3PIN
 
 endchoice # MPSL_CX_CHOICE
 
-endif	# MPSL_CX
+endif	# MPSL_CX || MPSL_CX_PIN_FORWARDER
```

Comment:
For line 61. we have select NRFX_GPIOTE, should we have this selet also for MPSL_CX_PIN_FORWARDER

Reply:
That's a good catch, thank you. I think the `select` should only work when `MPSL_CX` is enabled:
```
select NRFX_GPIOTE if MPSL_CX
```
What do you think?